code/modules/admin/verbs/request_internet_sound.dm

```diff
@@ -20,6 +20,15 @@
 		to_chat(usr, span_danger("Invalid URL. Please use a URL from one of the following sites: [replacetext(CONFIG_GET(string/request_internet_allowed), "\\", "")]"), confidential = TRUE)
 		return
 
+	var/credit = tgui_alert(usr, "Display who rquested the song? (Will show character name, NOTE if on the title screen it'll show CKEY instead)", "Credit Yourself?", list("No", "Yes", "Cancel"))
```

Comment:
```suggestion
	var/credit = tgui_alert(usr, "Credit yourself for requesting this song? (will show up as [usr.name])", "Credit Yourself?", list("No", "Yes", "Cancel"))
```

this is better than bringing up the edge case/exception to the whole character name/ckey thing as they can see what will be sent to the admin.